src/Dev/Tasks/TagsToShortcodeHelper.php

```diff
@@ -123,15 +127,18 @@ public function run()
                         continue;
                     }
 
-                    try {
-                        // Update table
-                        $this->updateTable($table, $field);
+                    if (!isset($tableList[strtolower($table)])) {
```

Comment:
That's just for PHPUnit. Not all tables get created when running tests, but the task will still try to run queries against them.

Originally we had a try/catch for this but that was hidding errors.